.vscode/settings.json

```diff
@@ -9,14 +9,21 @@
       "lib/**/*.js.map": true,
       "lib/**/*.js": true,
     },
-    "tslint.enable": true,
-    "tslint.run": "onType",
     "tslint.configFile": "tslint.json",
     "files.trimTrailingWhitespace": true,
     "editor.tabSize": 2,
     "editor.insertSpaces": true,
     "[typescript]": {
-      "editor.formatOnSave": true
+      "editor.formatOnSave": true,
+      "editor.defaultFormatter": "xaver.clang-format"
```

Comment:
Just to confirm - 

1) It seems like the goal of this change to update vscode settings to change default formatter for ts, cpp, and jsonc files. No other logic change. Correct ?

2) How do the MD files get formatted automatically too ?

Reply:
>> 1. the default formatter for .ts and .cpp are still clang-format. this change only add formatter support for .json and .md
>> 2. the indent, spaces, empty lines and alignment of table and so on